src/System.Net.Http.WinHttpHandler/src/project.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
     "Microsoft.Win32.Primitives": "4.0.0",
+    "System.Diagnostics.DiagnosticSource": "4.0.0-rc3-23729",
```

Comment:
I'm surprised this passes CI... I'd expect these 23729 versions numbers to cause failures due to the package verification.  I believe we're currently on 23808.